index.js

```diff
@@ -4,7 +4,7 @@ import MochaAdapter from './lib/adapters/MochaAdapter.js'
 import TapReporter from './lib/reporters/TapReporter.js'
 import ConsoleReporter from './lib/reporters/ConsoleReporter.js'
 import {Assertion, Test, Suite} from './lib/Data.js'
-import {autoRegister} from './lib/helpers.js'
```

Comment:
I have done this to be easier to export the whole `helpers`. For example if you want to use some functions to from the `helpers` to can easily import it, not to require every function like:
```js
import {createSuiteStart, createSuiteEnd, createTestStart, createTestEnd} from 'js-reporters/lib/helpers'
```
to be instead:
```js
import helpers from `js-reporters/lib/helpers`
```